src/components/UserList/Row.js

```diff
@@ -147,10 +149,28 @@ class Row extends Component {
       </View>
     );
   }
+  
+  onRequestCallback(result, error) {
+    if (error) {
+      Alert.alert('Error on Bookmark', error);
+    } else if (result) {
+      this.forceUpdate();
+    }
+  }
 
   sendRequest() {
     Actions.requestPage({ id: this.props.dataSource._id, me: this.props.dataSource.me });
   }
+  
+  setBookmark() {
+    if (this.state.bookmarked) {
+      UserUtil.bookmarkOff(this.onRequestCallback.bind(this), this.props.dataSource._id);
+      this.state.bookmarked = false;
+    } else {
+      UserUtil.bookmarkOn(this.onRequestCallback.bind(this), this.props.dataSource._id);
+      this.state.bookmarked = true;
```

Comment:
I guess yes~?
I mean thinking of scenario,

bookmarkOn 
->this.state.bookmarked = true 
->but bookmarkOn request fail

client bookmarked state === true but, server bookmarked state === still false, right?
Shoudn't we synchronize bookmarked state? or no?